cypress/integration/eholdings/notes.spec.js

```diff
@@ -1,24 +1,26 @@
+import { cy } from 'date-fns/locale';
+import { Accordion, Button, including, Link, Page, Pane } from '../../../interactors';
+
 describe('ui-eholdings: Notes', () => {
   before('logs in and navigates to eHoldings', () => {
+    cy.visit('/');
     cy.login('diku_admin', 'admin');
-    cy.visit('/eholdings');
+    cy.do(Link('eHoldings').click());
   });
 
   describe('creating a Note', () => {
     before('searching and opening provider', () => {
       cy.search('EBSCO');
-      cy.get('#search-results-content li:first-child').click();
+      cy.do(Pane('Providers').find(Link(including('EBSCO'))).click());
     });
 
     describe('clicking on note create button', () => {
       before(() => {
-        cy.get('#providerShowNotes #note-create-button').click();
+        cy.do(Accordion('Notes').find(Button('New')).click());
```

Comment:
Do we need to nest in the Accordion? Are there multiple `New` buttons on the screen?

Reply:
I prefer to leave Accordion, because it tells about the context. Even if the New button is uniq it's impossible to understand that the click does without taking a look to UI.

Reply:
I'm continue thinking in terms of bigtest and skip the describe text. In my opinon, tests would be better if they will be self documented.

Reply:
So, I can remove Accordion call if you think that it's unnecessary

Reply:
It's more of an art than a science about when to use nesting, but I tend to only refer to nesting if it is truly necessary to disambiguate. In this case though, `New` is pretty generic, so I see your point. It improves the readability of the test doesn't it?

Reply:
Yep, it does.